tests/saw/spec/handshake/rfc_handshake_tls13.cry

```diff
@@ -305,3 +305,61 @@ rfc2S2N msg = mkAct recordType messageType writer
                  | msg.sender == both then 'B'
                  else '!'
         noMessageType = TLS_HELLO_REQUEST // HelloRequests don't exist in tls1.3
+
+/**
+ * printHandshake and s2nToWords are two helper functions for debugging cryptol
+ * Take the counterexample from tls13rfcSimulatesS2N and plug the values into
+ * testParameters and testConnection. Run printHandshake with ascii set to on. 
+ * Note with ascii on, you may see actual_protocol_version = '"'. This is because
+ * decimal value 34 (hex value 22) == asci character double quotes. 
+ */
+
+testParameters : Parameters
+testParameters = {
+            compat_mode = [False, True],
+            psk_mode = False,
+            retry = False,
+            client_auth = True,
+            no_client_cert = True,
+            zero_rtt = False
+        }
+
+testConnection : connection
+testConnection = {
+            handshake = {handshake_type = 0x00000000,
+                message_number = 0x00000000},
+            mode = 0x00000001,
+            corked_io = '\NUL',
+            corked = zero,
+            is_caching_enabled = False,
+            resume_from_cache = False,
+            server_can_send_ocsp = False,
+            key_exchange_eph = False,
+            client_auth_flag = True,
+            actual_protocol_version = 0x22
+        }
+
+printHandshake : {len} (fin len, len >= 1) => ([3][8], [len][22][8], [3][8], [len][22][8])
```

Comment:
Are you able to create types here, something like `type messageDescription = [22][8]`?